Week3/moreVehicleToAdvertisementList.js

```diff
@@ -0,0 +1,17 @@
+//11. What if you add one more vehicle to the list, can you have that added to the advertisement without changing the code for question 10?
+
+vehicles = ['motorbike', 'caravan', 'bike', 'truck'];
+console.log(vehicles);
+vehicles.push('car');
+console.log(vehicles);
+
+let advertisedVehicles = '';
+
+for (let i = 0; i < vehicles.length; i++) {
+  if (i == vehicles.length - 1) {
```

Comment:
Ah, I forgot to check this one. It looks good but I think now it will result in:

`Amazing HappyMan's Garage, we service motorbikes, caravans, bikes, and trucks`.

But the `,` before the `and` is not correct:) can you make it so that the `,` is not added before the `and`?

Reply:
I found a solution for that as below; and it works. 


function newAdvertisement(advertisedVehicles, vehicles) {
  for (let i = 0; i < vehicles.length; i++) {
    if (i == vehicles.length - 1) {
      advertisedVehicles += 'and ' + vehicles[i] + 's.';
    } else if (i == vehicles.length - 2) {
      advertisedVehicles += vehicles[i] + 's ';
    } else {
      advertisedVehicles += vehicles[i] + 's, ';
    }
  }
  return advertisedVehicles;
}

console.log(newAdvertisement(advertisedVehicles, vehicles));